moveit_setup_assistant/moveit_setup_assistant/package.xml

```diff
@@ -27,6 +27,7 @@
   <depend>rclcpp</depend>
   <depend>moveit_setup_framework</depend>
   <depend>moveit_setup_srdf_plugins</depend> <!-- For collision_updater executable -->
+  <exec_depend>moveit_configs_utils</<exec_depend>
```

Comment:
```suggestion
  <exec_depend>moveit_configs_utils</exec_depend>
```

Reply:
Oops, that's what I get for using the GitHub web editor! Sorry.